irods/user.py

```diff
@@ -47,6 +55,13 @@ def remove(self):
     @property
     def members(self):
         return self.manager.getmembers(self.name)
+    
+    @property
+    def metadata(self):
+        if not self._meta:
+            self._meta = iRODSMetaCollection(
+                 self.manager.sess.metadata, User, self.name)
```

Comment:
Should this be UserGroup instead of User? Groups are just users without passwords in iRODS so this may just work, but it would be better to err on the side of consistency.

Reply:
Yes, we should have a test for both User and UserGroup, so we can defend our honor later.